pybaseball/datasources/fangraphs.py

```diff
@@ -0,0 +1,48 @@
+import lxml
+from typing import List
+
+import pandas
```

Comment:
I feel like `import pandas as pd` is a bit more normative / consistent?

Reply:
It is. I'll fix it.